File: problemSet1/cash.c
#include <cs50.h>
#include <stdio.h>

int calculate_coins(int cents);
int main(void)

{
    int cents;
    do
    {
        cents = get_int("Change owed: ");
    }
    while (cents <= 0);

    int result = calculate_coins(cents);
    printf("%i\n", result);
}

int calculate_coins(int cents)
{
    int counts = 0;

    while (cents > 0)
    {
        while (cents >= 25)
        {
            cents = cents - 25;
            counts++;
        }
        while (cents >= 10)
        {
            cents = cents - 10;
            counts++;
        }
        while (cents >= 5)
        {
            cents = cents - 5;
            counts++;
        }
        while (cents >= 1)
        {
            cents = cents - 1;
            counts++;
        }
    }
    return counts;
}

File: problemSet6/Sentimental Cash/cash.py
from cs50 import get_float


change = get_float("Change: ")

while change <= 0:
    change = get_float("Change: ")

cents = round(change * 100)
count = 0

while cents >= 25:
    cents -= 25
    count += 1

while cents >= 10:
    cents -= 10
    count += 1

while cents >= 5:
    cents -= 5
    count += 1

while cents >= 1:
    cents -= 1
    count += 1

print(count)
